extension/test/data_test.ts

```diff
@@ -62,5 +62,11 @@ describe('data.ts', function () {
         rcxDict.wordSearch('ぼんです', /* doNames= */ false)?.data
       ).to.not.include.something.like({ entry: /^凡打 .*/ });
     });
+
+    it('should return results for words with punctuation', function () {
+      expect(
+        rcxDict.wordSearch('ＤＶＤ-ＲＯＭ', /* doNames= */ false)?.data
+      ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
+    });
```

Comment:
I'm not sure if I should add more tests. I only wrote one test which covers the case mentioned in https://github.com/melink14/rikaikun/issues/1453.

Reply:
Judging from code coverage, I think we should also add a test for a word that contains a character less than 3000 that is not in the CONSTANT such that both halves of the conditional are testsed.